_projects/2022/100385774/100385774.Rmd

```diff
@@ -0,0 +1,902 @@
+---
+title: "Social Media Use in 2021"
+description: |
+  % of adults using each social media.
```

Comment:
Please give a longer description of the article. Tell the reader what is about to read. ;-)